connCheck undefined on v1.5.0

I receive this error when compile with v1.5.0

clickhouse.go:133:20: ch.conn.connCheck undefined (type *connect has no field or method connCheck)

--- a/connect_check_dummy.go
+++ b/connect_check_dummy.go
@@ -2,8 +2,6 @@
 
 package clickhouse
 
-import "net"
-
-func connCheck(conn net.Conn) error {
+func (conn *connect) connCheck() error {
 	return nil
 }

